sdks/java/managed/src/main/java/org/apache/beam/sdk/managed/Managed.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.sdk.managed;
+
+import com.google.auto.value.AutoValue;
+import java.util.Map;
+import java.util.regex.Pattern;
+import javax.annotation.Nullable;
+import org.apache.beam.sdk.schemas.transforms.SchemaTransform;
+import org.apache.beam.sdk.values.PCollectionRowTuple;
+
+public class Managed {
+  public static Read read() {
+    return new AutoValue_Managed_Read.Builder().setPattern(Read.PATTERN).build();
+  }
+
+  @AutoValue
+  public abstract static class Read extends SchemaTransform {
+    protected static final Pattern PATTERN =
+        Pattern.compile("beam:schematransform:org.apache.beam:[\\w-]+_read[\\w-]*:[\\w-]+");
+
+    abstract String getSource();
+
+    abstract Pattern getPattern();
```

Comment:
As cham mentions below, it's to get identifiers that match to a pattern (e.g. "_read" for `Managed.read()`). So users wouldn't be able to do something like `Managed.write().to(<kafka read>)`. 

> Pattern even a valid schema field type

Thanks for catching, yeah this would be problematic for cross-lang. Will make it a string here instead